pkgs/build-support/cc-wrapper/setup-hook.sh

```diff
@@ -70,12 +70,12 @@ ccWrapper_addCVars () {
     local role_post role_pre
     getHostRoleEnvHook
 
-    if [[ -d "$1/include" ]]; then
-        export NIX_${role_pre}CFLAGS_COMPILE+=" ${ccIncludeFlag:--isystem} $1/include"
```

Comment:
Why are we getting rid of `ccIncludeFlag`?

Reply:
PR message says

> Also remove ccIncludeFlag lookup, this was unused & not very useful.

Reply:
Oh oops, thanks.